### DISCUSSION TITLE
443 80已占用 如何使用Hy2

### DISCUSSION THREAD

**Discussion by usjinbao** [apernet/hysteria#713] [Q&A]
比如安装了nginx占用了443和80端口，此时想通过反向代理来实现如何解决？ 目前的问题是hy2端口改为其他端口的话 就会导致启动过程出错，无法正常启动

**Comment by mritd** [comment]
将监听端口(`listen: :2443`)改为其他端口, 不行的话上日志.

**Comment by tobyxdd** [comment]
而且 nginx 不开 QUIC 不应该占用 UDP 443

**Comment by yjd** [comment]
我一直跑的非443
